grade-scripts/grades-from-trackers.js

```diff
@@ -0,0 +1,118 @@
+const listManager = require('./shared/list-manager')
+const program = require('commander')
+const fs = require('fs')
+const execSync = require('child_process').execSync
+const chalk = require('chalk')
+
+const Grade = require('../src/classes/grade')
+const https = require('../src/https')
+const privacyPolicy = require('../src/privacy-policy')
+const scriptUtils = require('./shared/utils')
+
+program
+    .option('-i, --input <name>', 'The name to use when looking for sites, e.g. "test" will look in "test-sites"')
+    .option('-o, --output <name>', 'Output name, e.g. "test" will output files at "test-grades"')
+    .option('-f, --file <name>', 'Allow processing a subset of dumped site data, defined in a file')
+    .parse(process.argv)
```

Comment:
Would it be worth to add default options for ease of use, and maybe add to the PR a simple input text file like described in the README?
Minor, can be a followup